actix/src/sec-7-stream.md

```diff
@@ -1 +1,305 @@
-[WIP] 
\ No newline at end of file
+# Streams
+
+Actix allows the handling of Tokio Streams to act as a source of events
+or messages to actors. This is commonly used with networking servers
+and clients, for communication over TCP, UDP and UnixSockets.
+
+At it's core, Actix when using streams provides a mechanism for taking
```

Comment:
This is not correct.

If you take a look at `StreamHandler` it is only interface to handle `Stream` items, rather than converting its items to messages.

So to be accurate you can say that actix provides interface, similar to message handling, but for Stream's items (do note that it also allows you to handle errors and when stream ends which is different from regular messages)

Reply:
I think that is what I say though? I say that it it takes frames and then converts them to messages?

Reply:
You're not talking about `StreamHandler` here at all instead your future example is going to show how to hack around streams with a less effective way comparing to `StreamHandler`.

Aside from `StreamHandler` actix don't really has utilities to work with streams